packages/loaders/src/utils/spinner-coordinates.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export const strokeWidthFrames = {
+  0: 6,
+  28: 5,
+  31: 4,
+  36: 3,
+  48: 2,
+  56: 3,
+  68: 4,
+  80: 5,
+  91: 6
+};
+
+export const rotationFrames = {
+  0: 267,
+  1: -90,
+  28: -52,
+  31: -42,
+  33: -36,
+  35: -32,
+  36: -27,
+  38: -25,
+  40: -20,
+  41: -15,
+  43: -13,
+  45: -8,
+  46: -2,
+  48: 0,
+  50: 6,
+  51: 12,
+  53: 15,
+  55: 22,
+  56: 29,
+  58: 33,
+  60: 40,
+  61: 49,
+  63: 53,
+  65: 62,
+  66: 73,
+  68: 78,
+  70: 90,
+  71: 104,
+  73: 112,
+  75: 129,
+  76: 148,
+  78: 159,
+  80: 180,
+  81: 199,
+  83: 207,
+  85: 221,
+  86: 231,
+  88: 235,
+  90: 243,
+  91: 249,
+  93: 252,
+  96: 257,
+  98: 264
+};
+
+export const dasharrayFrames = {
+  0: 0,
+  10: 1,
+  25: 12,
+  30: 21,
+  35: 33,
+  36: 40,
+  38: 43,
+  40: 50,
+  41: 58,
+  43: 61,
+  45: 68,
+  46: 73,
+  48: 76,
+  50: 81,
+  58: 90,
+  68: 83,
+  70: 79,
+  75: 62,
+  76: 53,
+  78: 47,
+  80: 37,
+  81: 27,
+  83: 23,
+  85: 16,
+  90: 7,
+  98: 1
+};
```

Comment:
If these frames end up working well for us, we should revisit the Dots loader and pull keyframes from https://github.com/zendeskgarden/css-components/blob/jzempel/loaders/packages/loaders/src/_keyframes.css accordingly. May not be necessary for this PR, but should be done as a follow-on.

Reply:
I tried these and I'm getting some jitter towards the end I'm pretty sure this was the same on my values too. See [test case](https://jsbin.com/kecosapiye/1/edit?js,output).

![kapture 2018-10-15 at 16 02 30](https://user-images.githubusercontent.com/143402/46930826-c5acb480-d093-11e8-849c-76fdcc71b512.gif)

Seems the rotation linear progression is too slow for the dasharray linear changes any ideas?

Reply:
Along with modifying the diff calculation in `computeFrames`, I did a fair bit of fussing with the keyframe values. I'm getting much better results with the following:

```js
const ROTATION_FRAMES = {
  0: -90,
  8: -81,
  36: -30,
  41: -18,
  44: -8,
  48: 0,
  55: 22,
  63: 53,
  64: 62,
  66: 67,
  68: 78,
  69: 90,
  71: 99,
  73: 112,
  74: 129,
  76: 138,
  78: 159,
  79: 180,
  81: 190,
  83: 207,
  84: 221,
  86: 226,
  88: 235,
  90: 243,
  99: 270
};

const DASHARRAY_FRAMES = {
  0: 0,
  13: 2,
  26: 13,
  53: 86,
  58: 90,
  63: 89,
  64: 88,
  66: 86,
  68: 83,
  69: 81,
  71: 76,
  73: 70,
  74: 62,
  76: 58,
  78: 47,
  79: 37,
  81: 31,
  83: 23,
  84: 16,
  88: 10,
  89: 7,
  98: 1,
  99: 0
};
```

See modified [test case](https://jsbin.com/facogisudu/edit?js,output).

Reply:
Result...

![spinner](https://user-images.githubusercontent.com/143773/46977309-b8d29580-d080-11e8-95b1-8be26c93972f.gif)